apps/src/templates/feedback/ParticipantFeedbackNotification.story.jsx

```diff
@@ -1,20 +1,26 @@
 import React from 'react';
+import {Provider} from 'react-redux';
+import {reduxStore} from '@cdo/storybook/decorators';
 import ParticipantFeedbackNotification from './ParticipantFeedbackNotification';
 import sinon from 'sinon';
 
-export default storybook => {
-  return storybook
-    .storiesOf('ParticipantFeedbackNotification', module)
-    .withReduxStore()
-    .addStoryTable([
-      {
-        name: 'ParticipantFeedbackNotification',
-        story: () => {
-          withFakeServer();
-          return <ParticipantFeedbackNotification studentId={123} />;
-        }
-      }
-    ]);
+export default {
+  title: 'ParticipantFeedbackNotification',
+  component: ParticipantFeedbackNotification
+};
+
+const Template = args => {
+  withFakeServer();
+  return (
+    <Provider store={reduxStore()}>
+      <ParticipantFeedbackNotification {...args} />
+    </Provider>
+  );
+};
+
+export const ParticipantFeedbackNotificationStudent = Template.bind({});
```

Comment:
Nit: It might improve readability to remove 'ParticipantFeedbackNotification' from the name 'ParticipantFeedbackNotificationStudent' since the story name gets nested under the title. Currently, I think that would be "Participant Feedback Notification > Participant Feedback Notification Student". Not a big deal either way, but might improve readability.